server/cmwell-batch/src/main/scala/cmwell/batch/boot/Runner.scala

```diff
@@ -48,6 +52,8 @@ object Runner extends App with LazyLogging{
 //  val isIndexer:Boolean =  java.lang.Boolean.valueOf(Option(System.getProperty("isIndexer")).getOrElse("false"))
 
   logger info ("Running batch")
+  SysOutOverSLF4J.sendSystemOutAndErrToSLF4J()
```

Comment:
add a comment to let the reader know this MUST come after **some** log message, and thus come directly after the `"Running batch"` log.